docs/whats-new-in-c8/whats-new-in-c8.md

```diff
@@ -68,10 +67,17 @@ Cylc has been re-architected to support a remote web UI:
   - command integration, for interacting with task, jobs, and schedulers
 - a new terminal UI (TUI) as part of the CLI
 
-![Cylc Hub](./img/hub.png)
-![Cylc UI dash](./img/cylc-ui-dash.png)
-![Cylc UI tree](./img/cylc-ui-tree.png)
-![Cylc TUI](./img/cylc-tui.png)
+![title Cylc Hub](./img/hub.png)
```

Comment:
I think that a bit more label, and something for text readers to get hold of might be handy.